.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: node_js
+
+node_js:
+  - "node"
+
+script:
+  - yarn test
```

Comment:
Leave a blank line at the bottom. It's needed in a few other files.

Reply:
Done :+1: